en/mkdocs.yml

```
@@ -87,6 +87,8 @@ nav:
           - Introduction to Chat Agents: learn/ai/agents/introduction-to-chat-agents.md
           - Introduction to Inline Agents: learn/ai/agents/introduction-to-inline-agents.md
           - Integrating Agents with External Endpoints: learn/ai/agents/integrating-agents-with-external-endpoints.md
+        - RAG:
+          - Build a RAG Application with Ballerina Integrator: learn/ai/rag/build-a-rag-application.md
```

```suggestion
          - Build a RAG Application: learn/ai/rag/build-a-rag-application.md
```
We can drop the with BI part

---

Ack